plugin/pkg/admission/persistentvolume/label/admission.go

```diff
@@ -75,6 +77,13 @@ func (l *persistentVolumeLabel) Admit(a admission.Attributes) (err error) {
 		}
 		volumeLabels = labels
 	}
+	if volume.Spec.GCEPersistentDisk != nil {
+		labels, err := l.findGCEPDLabels(volume)
+		if err != nil {
+			return admission.NewForbidden(a, fmt.Errorf("error querying GCE PD volume %s: %v", volume.Spec.GCEPersistentDisk.PDName, err))
```

Comment:
we can put the "dummy" constant in pkg/volume (as opposed to a plugin) and use that in admission and the plugins themselves.

it is leaky as hell but is far less work than adding plugins and a framework to the API server.  Should we consider it as a short term fix until we come up with a better way?